appPackage/manifest.json

```diff
@@ -75,9 +76,9 @@
                 "personalTab",
                 "channelTab"
             ], 
-            "name": "Home", 
-            "contentUrl": "http://localhost:3000/", 
-            "websiteUrl": "http://localhost:3000/"
+            "name": "Feed", 
+            "contentUrl": "https://ashy-pond-00441c81e.5.azurestaticapps.net/", 
```

Comment:
We don't really want these hard coded, as Production will point to test, etc.

We want it to use `localhost:3000` / `<testurl>` / `<produrl>` etc.